src/plugins/intel_gpu/src/graph/impls/onednn/convolution_onednn.cpp

```diff
@@ -318,25 +318,17 @@ struct convolution_onednn : typed_primitive_onednn_impl<convolution> {
             _attrs->set_zero_points(DNNL_ARG_WEIGHTS, 0, dnnl::memory::dims{}, _wzp_data_type);
         }
 
-        if (zero_bias) {
-            auto prim_desc = std::make_shared<dnnl::convolution_forward::primitive_desc>(
-                                    ib.get_engine().get_onednn_engine(),
-                                    dnnl::prop_kind::forward_inference, dnnl::algorithm::convolution_direct,
-                                    input_md, weights_md, output_md,
-                                    strides, dilates, padding_l, padding_r,
-                                    *_attrs.get());
-            _pd = *prim_desc;
-        } else {
-            auto bias_md = onednn::layout_to_memory_desc(impl_params->get_input_layout(2), dnnl::memory::format_tag::any, true);
-            auto prim_desc = std::make_shared<dnnl::convolution_forward::primitive_desc>(
-                                    ib.get_engine().get_onednn_engine(),
-                                    dnnl::prop_kind::forward_inference, dnnl::algorithm::convolution_direct,
-                                    input_md, weights_md, bias_md, output_md,
-                                    strides, dilates, padding_l, padding_r,
-                                    *_attrs.get());
-            _pd = *prim_desc;
+        dnnl::memory::desc bias_md = nullptr;
+        if (!zero_bias) {
+            bias_md = onednn::layout_to_memory_desc(impl_params->get_input_layout(2), dnnl::memory::format_tag::any, true);
         }
-
+        auto prim_desc = std::make_shared<dnnl::convolution_forward::primitive_desc>(
+                                ib.get_engine().get_onednn_engine(),
+                                dnnl::prop_kind::forward_inference, dnnl::algorithm::convolution_direct,
+                                input_md, weights_md, bias_md, output_md,
+                                strides, dilates, padding_l, padding_r,
+                                *_attrs.get());
```

Comment:
The behavior seems exactly same as below. Am I missing something?

Reply:
I doubt the issue is hidden by compilation result change, but the real issue is unknown..